toml.go

```diff
@@ -71,18 +71,14 @@ func (t *Tree) Keys() []string {
 }
 
 // Get the value at key in the Tree.
-// Key is a dot-separated path (e.g. a.b.c).
+// Key is a dot-separated path (e.g. a.b.c) without single/double quoted strings
```

Comment:
Missing `.` at the end of the sentence. Also, do you mind adding a comment about using `GetPath` if the support for non-bare keys is required?